### DISCUSSION TITLE
video card NVidia P106-100

### DISCUSSION THREAD

**Discussion by AlDo2358** [lllyasviel/Fooocus#3061] [Q&A] [answered]
To the developers, thank you for the wonderful Fooocus.
Please tell me how to make fooocus see the video card. Fooocus does not see the Nvidia video card. Windows is switched to it and all applications and toys see it and work with it. Below are the screenshots.
![1pic](https://github.com/lllyasviel/Fooocus/assets/158803611/0fe8c6e8-b913-4e1b-8767-5b8abc7df69f)
![2pic](https://github.com/lllyasviel/Fooocus/assets/158803611/cae9cc20-0e97-4dab-90b2-238b68f6a76d)

**Comment by mashb1t** [comment]
P106-100 is a mining card from 2017 and has reached end of life. It only supports CUDA until 6.1, Fooocus (and many other modern AI tools) needs 11.x/12.x.
=> This card is not supported, sorry.

**Comment by AlDo2358** [comment]
Good afternoon. My improvement has taken a turn for the worse. In an attempt to improve performance reinstalled windows 10 to windows 10 pro 2020.  Everything is now running 30% faster but fooocus didn't want to work. On pc with amd 5700 xt it worked, slow but it worked. A picture 1024x1024 was formed in 30-40 minutes. Now there is an error. I don't want and have no right to take up your time, at least give me a hint or two words on how to run fooocus. Thanks.
![Снимок](https://github.com/lllyasviel/Fooocus/assets/158803611/7a07dca2-363b-474a-aa55-842f4a4f2fb9)

**Reply by mashb1t** [reply, marked as answer]
Sure, please double-check your swap setup, see https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#system-swap.
This is most likely the issue.

**Comment by AlDo2358** [comment]
Thank you. Everything works. With the same assignment, the picture is formed:
CPU Ryzen 7 5700G:
P106-100 - Bat file for NVidia - 21 min.
ATI Radeon integrated graphics - Bat file for AMD - 28 minutes.
CPU Xeon4620:
ATI Radeon RX 5700 XT - Bat file for AMD- 34 minutes

**Comment by AlDo2358** [comment]
Good afternoon.
Please check, it might be bugs.
1. After image generation is finished, fooocus does not free memory and video memory. PC hangs. See figure 1,2,3.
2. Skip button does not work in the browser. It is not possible to cancel.
3. Browser window with fooocus hangs, but fooocus terminated correctly.
In any - Opera, Brave, Firefox,
4. The fooocus settings are not saved in the browser window. I have to repeat them every time after a hang.

Fooocus is still the best.
![pic1](https://github.com/user-attachments/assets/a54b0ceb-8d66-4341-a06c-80984401bed0)

**Reply by mashb1t** [reply]
1. this is the correct behavior
2. works for me in both local and Colab environments, must be your PC being overloaded or your PC taking too long for unskippable parts to process
3. same as my 2nd answer
4. keep the window open and do not close it. If Fooocus ceashes or you close its terminal window, the Web GUI displays a "Reconnect" button. Reatart Fooocus using the terminal / run.bat manually and then click "Reconnect" to not loose any settings.

As mentioned in a previous comment, your GPU has reached end of life and is a mining card => not optimized for AI workload.

All the best for your further testing!

**Comment by AlDo2358** [comment]
Good afternoon.
Installed 64GB of RAM and a Ryzen 7.
Fooocus is happy, works fast and comfortable. Thanks.

Using Help, gave a task, but it is not working for me. Sending screenshots. Did I do something wrong or fooocus can't do it?
Pic1 - task.
Pic2 and Pic3 are the result of execution.
![pic3](https://github.com/user-attachments/assets/5b07bb02-a951-4391-a5b5-93b7f2096844)
![pic2](https://github.com/user-attachments/assets/4baa7dab-3293-4f24-9ff7-5a4097dd7239)
![pic1](https://github.com/user-attachments/assets/78398c64-9e38-4652-8ac2-c4adf4b10b56)
